.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+
+# One line description
+
+Please include a paragraph here describing what this PR does:
+what kind of change (refactoring, bugfix, feature), what is changed
+(short summary of new features or fix).
+
+If bugs are addressed in this PR, include the issue numbers.
+
+# Who benefits?
+
+Axom team, "LLNL customers" if you can't mention in detail,
+specific LLNL projects if you can, or outside feature request.
+
+# Design review
+
+If this is a big change, describe design review meetings:
```

Comment:
This part should probably also discuss API changes.

Reply:
Minor wording:
i think this is more of a brief design outline vs a design review? 
The PR itself is part of the review?

Reply:
You're right, @cyrush --- the wording probably needs revision.  This section of the PR template is here mainly for large API changes or additions, that would need a design review meeting.  The intent is to prominently show that we *had* a design review and to *(briefly)* summarize the main points.  See also the discussion attached to line 15.